test/functional/legacy-fixtures/regression/legacy-gh-16/testcafe-fixtures/index.test.js

```diff
@@ -0,0 +1,15 @@
+'@fixture GH-16';
+'@page ./index.html';
+
+
+'@test'['Check stored data'] = {
+    '1.Set data and reload iframe': inIFrame('#iframe', function () {
+        this.data = 200;
+
+        act.click('#replaceSrc');
+    }),
+
+    '2.Check data': function () {
+        ok(this.data.toString(), '200');
```

Comment:
It was in original test and it caused to a problem ([reason](https://github.com/DevExpress/testcafe-legacy-api/issues/16#issuecomment-250688729))